### PR TITLE
This is a comment.

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @andrew-scoppa
- [ ] @Chembros123
- [ ] @IDNRMMLRD
- [ ] @MRKnepler
- [ ] @WallyHartshorn
- [ ] @bfmonroe
- [ ] @muzotabag
- [ ] @timp70291

### PR DESCRIPTION
**Pull request by timp70291**
Made changes to file.

This PR closes #2.

### CODE REVIEW

File: _slides/09-timp70291.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Tim was here!
```
Comment (andrew-scoppa) [inline _slides/09-timp70291.md:6]: This funny - have you considered markdown

File: _slides/09-timp70291.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (muzotabag) [inline _slides/09-timp70291.md:7]: Nice

File: _slides/09-timp70291.md (hunk 1 above)
Comment (andrew-scoppa) [reply _slides/09-timp70291.md:6]: Any other suggestions?

Comment (timp70291) [reply _slides/09-timp70291.md:6]: This is a reply from Tim

**Review by andrew-scoppa — APPROVED** [review]
I've approved your PR

File: _slides/09-timp70291.md (hunk 2 above)
Comment (timp70291) [reply _slides/09-timp70291.md:7]: This is a reply from Tim

**Comment by WallyHartshorn** [comment]
The Internet will never be the same. Nice job.

**Pull request merged**